src/lipalightninglib.udl

```diff
@@ -85,6 +85,9 @@ interface LightningNode {
     [Throws=LnError]
     sequence<FailedSwapInfo> get_unresolved_failed_swaps();
 
+    [Throws=LnError]
+    FailedSwapRefundInfo prepare_refund_failed_swap(FailedSwapInfo failed_swap_info, string to_address, u32 onchain_fee_rate);
```

Comment:
Might make sense to expand the `FailedSwapRefundInfo` to be used as an argument in the `resolve_failed_swap` method.

Reply:
I think we don't need to force the use of the prepare method. Why do we need more info other than the swap address?

Reply:
I was just thinking from the client perspective, but I might have misunderstood the use of the `prepare_refund_failed_swap` method.

As a client if, and here I'm not sure if that's the case, I always call the `prepare_refund_failed_swap` method before refunding a swap would prefer to just supply the response from this to the `refund_failed_swap` method.
But if the client doesn't always call the prepare method before, this doesn't make sense.

Reply:
I was just thinking from the client perspective, but I might have misunderstood the use of the prepare_refund_failed_swap method.

As a client if, and here I'm not sure if that's the case, I always call the prepare_refund_failed_swap method before refunding a swap would prefer to just supply the response from this to the refund_failed_swap method.
But if the client doesn't always call the prepare method before, this doesn't make sense.